tests/resources/test_resource_utils.py

```diff
@@ -0,0 +1,54 @@
+from unittest.mock import patch
+
+from gnomad.resources import resource_utils
+
+
+class TestTableResource:
+    @patch("hail.read_table")
+    def test_read_table(self, read_table):
+        resource = resource_utils.TableResource("gs://my-bucket/table.ht")
+
+        ds = resource.ht()
+        read_table.assert_called_with("gs://my-bucket/table.ht")
+        assert ds == read_table.return_value
+
+
+class TestMatrixTableResource:
```

Comment:
I might not be understanding the docs for pytest correctly, but are the tests below checked? How would these classes be checked?

Reply:
Pytest automatically discovers tests in `testpaths` directories configured in pytest.ini.

> * In those directories, search for `test_*.py` or `*_test.py` files, imported by their test package name.
> * From those files, collect test items:
>    * `test` prefixed test functions or methods outside of class
>    * `test` prefixed test functions or methods inside `Test` prefixed test classes (without an `__init__` method)

https://docs.pytest.org/en/6.2.x/goodpractices.html#test-discovery